fulp_modules/features/antagonists/monster_hunter/hunting_contract.dm

```diff
@@ -39,23 +42,22 @@
 	if(weapons.len)
 		for(var/datum/hunter_weapons/contraband as anything in weapons)
 			data["items"] += list(list(
-			"id" = contraband.type,
-			"name" = contraband.name,
-			"desc" = contraband.desc,
+				"id" = contraband.type,
+				"name" = contraband.name,
+				"desc" = contraband.desc,
 			))
-	var/check_completed = TRUE  ///determines if all objectives have been completed
 	if(owner)
-		for(var/datum/objective/obj as anything in owner.objectives)
-			data["objectives"] += list(list(
-			"explanation" = obj.explanation_text,
-			"completed" = (obj.check_completion()),
-			))
-			if(!(obj.check_completion()))
-				check_completed = FALSE
-		data["all_completed"] = check_completed
-		data["number_of_rabbits"] = owner.rabbits_spotted
```

Comment:
i dont like that this is removed, it served to keep track of how many rabbits the player found if u can re-add it and put it somewhere on the button tooltip tht would be nice